### PR TITLE
add sdk

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

add `window.courierSdk` and `window.courierAsyncInit` array

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/components/README.md [hunk 1]
```diff
@@ -34,3 +36,68 @@ Installation is simple. All you need to do is add `<courier>` components to your
...
+window.courierConfig = {
+  components: {
+    toast: {
+      autoClose: false,
+    },
+  },
+};
+```
+
+> Inline configuration attributes will take precedence over `window.courierConfig` options
```
Comment (tk26) [inline packages/components/README.md:103]: maaaaybe having this note above before the examples is more helpful.

File: packages/components/src/index.tsx [hunk 2]
```diff
@@ -1,58 +1,157 @@
...
+    ...getAttrsAsJson(inboxElement),
+  };
+
+  const toastElement = document.querySelector("courier-toast") ?? undefined;
+  const toastConfig = {
+    ...componentConfigs?.toast,
+    ...getAttrsAsJson(toastElement),
+  };
+
+  console.log("toastConfig", toastConfig);
```
Comment (tk26) [inline packages/components/src/index.tsx:120]: maybe remove

**Review by tk26 — APPROVED** [review]

File: packages/components/src/index.tsx (hunk 2 above)
Comment (rileylnapier) [reply packages/components/src/index.tsx]: updated

File: packages/components/README.md (hunk 1 above)
Comment (rileylnapier) [reply packages/components/README.md]: updated

**Review by tk26 — DISMISSED** [review]

File: packages/components/README.md [hunk 3]
```diff
@@ -34,3 +36,68 @@ Installation is simple. All you need to do is add `<courier>` components to your
   <script src="https://courier-components-xvdza5.s3.amazonaws.com/latest.js"></script>
 </body>
 ```
+
+## [SDK](#SDK)
```
Comment (cgradwohl) [inline packages/components/README.md:40]: nice!

**Review by cgradwohl — APPROVED** [review]

**Review by tk26 — APPROVED** [review]

File: packages/components/README.md (hunk 3 above)
Comment (RoyalSix) [reply packages/components/README.md:40]: bitmoji

**Pull request merged**